android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.vydia.RNUploader">
 
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <uses-permission android:name="android.permission.CHANGE_NETWORK_STATE" />
```

Comment:
these were added previously to listen to network changes. Thanks to Worker features, these are no longer required.